dist/main.js.LICENSE.txt

```diff
@@ -1,8 +0,0 @@
-/**
```

Comment:
![Capture0](https://user-images.githubusercontent.com/93039359/206868746-48846388-6a8a-4fdb-80ac-4da62de213b2.PNG)

- [ ] It's great that you have added a descriptive readme but i guess this link is not working 😨 It's because you don't hav e a license file in your repository 🤔 It would be nice if you add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍